contrib/scripts/ast_coredumper

```diff
@@ -411,7 +458,7 @@ find_pid() {
 	# Now that we have the pids, let's get the command and
 	# its args. We'll add them to an array indexed by pid.
 	declare -a candidates
-	while read LINE ; do
+	while read -r LINE ; do
```

Comment:
Could potentially avoid the regex with:

    while read -r pid prog args ; do

Reply:
I did it this way so if for some reason the result didn't match the regex, we'd skip that line and continue otherwise we won't be sure that we got a valid line.